### DISCUSSION TITLE
Failing calls after update?

### DISCUSSION THREAD

**Discussion by ndkazu** [Fair-Squares/fair-squares#260] [General]
I just did a full run of the guide using the main branch, and everything is working

**Comment by cuteolaf** [comment]
Did you test the flow on this branch?
https://github.com/Fair-Squares/fair-squares/tree/add-account_id-tenant-struct

**Comment by cuteolaf** [comment]
The commented calls were failing on the master branch as well.

**Reply by ndkazu** [reply]
Everything is succeeding when I do it on the latest main branch here....no call is failing.
 strange

**Reply by cuteolaf** [reply]
hmm... could you please try on this branch?
https://github.com/Fair-Squares/fair-squares/tree/add-account_id-tenant-struct

**Reply by ndkazu** [reply]
ok, give me 10~20min

**Reply by ndkazu** [reply]
for now, it's working fine. waiting for the investor's vote to be enacted

**Reply by ndkazu** [reply]
Successfully voted for the representative session. Only 2 calls left.

**Comment by ndkazu** [comment]
Everything works on both branches man....wait I still have the guaranty deposit payment

**Reply by ndkazu** [reply]
Yup, every call is working fine on both branches...
rent paid & redistributed to owners.

**Comment by cuteolaf** [comment]
It's so weird.

**Reply by ndkazu** [reply]
some calls are spread over 2 pages in the document: make sure you copy them correctly. Maybe that's the cause?

**Reply by ndkazu** [reply]
lunch time for me!
